NeoML/include/NeoML/Dnn/Dnn.h

```diff
@@ -700,6 +686,56 @@ inline CArchive& operator>>( CArchive& archive, CDnn& dnn)
 
 void NEOML_API SerializeLayer( CArchive& archive, IMathEngine& mathEngine, CPtr<CBaseLayer>& layer );
 
+//------------------------------------------------------------------------------------------------------------
+
+// Result of CReferenceDnnFactory::CreateReferenceDnn
+class NEOML_API CDnnReference : public IObject {
+public:
+	CDnn Dnn;
+
+	static CPtr<CDnnReference> Create( CRandom& random, IMathEngine& mathEngine )
+	{ return new CDnnReference( random, mathEngine ); }
```

Comment:
Should this one be public? Dnn is not initialized here

Reply:
fixed